### PR TITLE
[TACHYON-1428] Documented parts of the different modules in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: clients/unshaded/src/main/java/tachyon/client/file/FileSystemMasterClient.java [hunk 1]
```diff
@@ -364,6 +364,7 @@ public Boolean call() throws TachyonTException, TException {
    * Schedules the async persistence of the given file.
    *
    * @param fileId the file id
+   * @return RPC result of the scheduling
```
Comment (aaudiber) [inline clients/unshaded/src/main/java/tachyon/client/file/FileSystemMasterClient.java]: I think the `Void` in the method signature here is supposed to be lowercase, no callers use the return value.

File: clients/unshaded/src/main/java/tachyon/hadoop/HdfsFileInputStream.java [hunk 2]
```diff
@@ -308,6 +310,10 @@ public void readFully(long position, byte[] buffer) throws IOException {
   /**
    * This method is not supported in {@link HdfsFileInputStream}.
    *
+   * @param position N/A
+   * @param buffer N/A
+   * @param offset N/A
+   * @param length N/A
    * @throws IOException always
    */
   @Override
```
Comment (aaudiber) [inline clients/unshaded/src/main/java/tachyon/hadoop/HdfsFileInputStream.java]: should `seek` below be documented?

File: underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java [hunk 3]
```diff
@@ -143,10 +143,13 @@ public OutputStream create(String path, short replication, int blockSizeByte) th
...
-   * recursive will delete all objects with given prefix.
-   * parent will not be deleted.
-   * Method always returns {@code true}.
+   * Parent will not be deleted.
+   *
+   * @param path the file or folder name
+   * @param recursive will delete all objects with given prefix
+   * @return always returns {@code true}
+   * @throws IOException if a non-Tachyon error occurs
+   * @see UnderFileSystem#delete(java.lang.String, boolean)
```
Comment (aaudiber) [inline underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java]: with the `@inheritDoc` we shouldn't need this. Same for mkdirs and rename below

File: underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java [hunk 4]
```diff
@@ -291,7 +302,7 @@ private String makeQualifiedPath(String path) {
   /**
    * @inheritDoc
    *
-   * @see tachyon.underfs.UnderFileSystem#rename(java.lang.String, java.lang.String)
+   *
```
Comment (aaudiber) [inline underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java]: extra blank line

File: underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java [hunk 5]
```diff
@@ -300,6 +311,12 @@ private String makeQualifiedPath(String path) {
    * For each returned object: src in the name is replaced with dst.
    * Object is deleted if copy was successful.
    * Directory rename always returns true.
+   *
+   * @param src The source file or folder name
```
Comment (aaudiber) [inline underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java]: lowercase `@param` sentence

File: underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java [hunk 6]
```diff
@@ -300,6 +311,12 @@ private String makeQualifiedPath(String path) {
...
+   * @param dst The destination file or folder name
+   * @return true if succeed, false otherwise
```
Comment (aaudiber) [inline underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java]: true if rename succeeds, false otherwise

**Comment by aaudiber** [comment]
Jenkins, test this please

**Comment by aaudiber** [comment]
Thanks for the PR! I've left some comments

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comments. I updated the PR accordingly.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**